asset-managment/niapm.yml

```diff
@@ -0,0 +1,1405 @@
+        type: integer
+        example: 2
+      systemName:
+        description: Hostname of the system.
+        type: string
+        example: APM-PXI1
+      state:
+        $ref: '#/definitions/AssetState'
+  ExternalCalibrationOperator:
+    title: External Calibration Operator
+    description: Specifies the operator that performed the calibration.
+    type: object
+    properties:
+      displayName:
+        description: Display name of logged in user.
+        type: string
+      userId:
+        description: Identifier of the logged in user.
+        type: string
+  SelfCalibrationModel:
+    title: Self Calibration
+    description: Contains data from the last self calibration.
+    type: object
+    required:
+        - date
+    properties:
+      temperatureSensors:
+          description: An array of temperature sensor information.
+          type: array
+          items:
+            $ref: '#/definitions/TemperatureSensor'
+      isLimited:
+        description: Specifies whether the last internal/self calibration of the asset was a limited calibration.
+        type: boolean
+        example: false
+      date:
+        description: ISO-8601 formatted timestamp specifying the last date the asset was self-calibrated.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+  ExternalCalibrationBaseModel:
+    title: Base External Calibration
+    description: Contains data for an external calibration entry.
+    type: object
+    required:
+        - date
+        - recommendedInterval
+        - nextRecommendedDate
+        - temperatureSensors
+    properties:
+      temperatureSensors:
+          description: An array of temperature sensor information.
+          type: array
+          items:
+            $ref: '#/definitions/TemperatureSensor'
+      isLimited:
+        description: Specifies whether the last external calibration of the asset was a limited calibration.
+        type: boolean
+        example: false
+      date:
+        description: ISO-8601 formatted timestamp specifying the last date the asset was externally calibrated.
+        type: string
+        format: iso-date-time
+        example: '2018-05-07T18:58:05.219692Z'
+      recommendedInterval:
+        description: Specifies the manufacturer-recommended calibration interval in months.
+        type: integer
+        example: 12
+      nextRecommendedDate:
+        description: ISO-8601 formatted timestamp specifying the recommended date for the next external calibration.
+        type: string
+        format: iso-date-time
+        example: '2019-05-07T18:58:05.219692Z'
+      comments:
+        description: Specifies any calibration comments.
+        type: string
+  ExternalCalibrationModel:
+    allOf:
+      - $ref: '#/definitions/ExternalCalibrationBaseModel'
+      - title: External Calibration
+      - description: Contains data from the last external calibration.
```

Comment:
"Values from the last external calibration." or "Data from the last external calibration."?